src/Framework/Runner/Runner.cs

```
@@ -428,6 +426,20 @@ public Runner(IRunnerSetupData setupData)
             IsTesting = setupData.IsTesting;
             ExcludedCategory = setupData.ExcludedCategory;
 
+            if(!string.IsNullOrEmpty(setupData.AdditionalResolutionDirectories))
+            {
+                var splits = setupData.AdditionalResolutionDirectories.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries);
+                if (!splits.Any()) return;
```

It is better to throw an exception to let users know there is an error